corehq/apps/app_manager/templates/app_manager/partials/mobile_report_configs.html

```diff
@@ -60,7 +60,22 @@
                             <input type="text" data-bind="value: display">
                         </td>
                         <td>
-                            <input type="text" data-bind="value: description">
+                            <p>
```

Comment:
move underneath text box please (it's a less common action, for more advanced users)